platform/ui/src/contextProviders/DialogProvider.js

```diff
@@ -118,6 +148,8 @@ const DialogProvider = ({ children, service }) => {
         contentProps,
         position,
         defaultPosition,
+        centralize = false,
```

Comment:
nix, and use center as position if none is provided for `position` or `defaultPosition`? Could also allow `defaultPosition` to support `string` values like `left`, `center`, `topRight`, if you felt it was important.

Reply:
Just curious, do we need `defaultPosition`? Do dialogs need to be able to return to an initial position?

For example:

- position: `null`; default to center
- position: `{ x: 200, y: 200 }`; initial
    - When dialog is closed: `{ position: { x, y } }`
    - Optionally use those values when opening same dialog if you want to preserve
    - OR, if it's draggable, track internally and key off `preservePosition` flag

Reply:
If we want to open the dialog directly on top of a tool which is a very common the defaultPosition would be very handy.

Reply:
Wouldn't we just set `position`?

Reply:
The defaultPosition will be set only once, position can be used for aync positioning (internally now). I have updated the code so that we only expose defaultPosition and use position only to centralize the component internally (async task since we want to calculate based on the rendered element).